helm/istio/values.yaml

```diff
@@ -0,0 +1,70 @@
+# Default values for Istio.
+
+##
+manager:
+  replicaCount: 1
+  image:
+    repository: docker.io/istio/manager
+    tag: 0.1.5
```

Comment:
Should the image values be kept in sync with https://github.com/istio/istio/blob/master/istio.VERSION and generated by updateVersion? https://github.com/istio/istio/tree/master/tests#updateversionsh